cmd/operator/install.go

```diff
@@ -2,6 +2,8 @@ package operator
 
 import (
 	"context"
+	"embed"
+	_ "embed"
```

Comment:
Duplicate?

Reply:
The first one is to import "embed" package and the other one is to initialize 
```
//go:embed manifest.yaml
```